appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/node/LeafNode.kt

```diff
@@ -0,0 +1,22 @@
+package com.bumble.appyx.navigation.node
+
+import com.bumble.appyx.interactions.core.plugin.Plugin
+import com.bumble.appyx.interactions.permanent.PermanentAppyxComponent
+import com.bumble.appyx.navigation.modality.NodeContext
+
+open class LeafNode(
+    nodeContext: NodeContext,
+    view: NodeView = EmptyNodeView(),
+    plugins: List<Plugin> = listOf(),
+) : Node<Nothing>(
+    appyxComponent = PermanentAppyxComponent(
```

Comment:
this means that this Node can have children? Yes, we don't have access to this model but is it conceptually correct?

Reply:
good catch, replacing with `EmptyAppyxComponent`